api/types.go

```diff
@@ -67,7 +67,8 @@ type ChatResponse struct {
 	CreatedAt time.Time `json:"created_at"`
 	Message   Message   `json:"message"`
 
-	Done bool `json:"done"`
+	LoadedMessages []Message `json:"loaded_messages,omitempty"`
```

Comment:
How can we handle `MESSAGE` in the `/generate` endpoint?

Reply:
I changed this to the be returned in the `/api/show` endpoint instead, so you'll have to call that endpoint first to get the messages.